static/app/views/replays/detail/detailLayout.tsx

```diff
@@ -70,17 +60,32 @@ function DetailLayout({children, event, orgId, crumbs}: Props) {
               ]}
             />
           </FeedbackButtonWrapper>
-          {eventRow}
+          <React.Fragment>
+            <Layout.HeaderContent>
+              <EventHeader event={event} />
+            </Layout.HeaderContent>
+            <MetaDataColumn>
+              <EventMetaData event={event} crumbs={crumbs} />
+            </MetaDataColumn>
+          </React.Fragment>
         </Layout.Header>
         {children}
       </React.Fragment>
     </SentryDocumentTitle>
   );
 }
 
+const HeaderPlaceholder = styled(function HeaderPlaceholder(
+  props: React.ComponentProps<typeof Placeholder>
+) {
+  return <Placeholder width="100%" height="19px" {...props} />;
+})`
+  background-color: ${p => p.theme.background};
+`;
```

Comment:
TIL: inline the component to the styled call

Lets see what the fat-arrow syntax looks like in there:
```suggestion
const HeaderPlaceholder = styled((props: React.ComponentProps<typeof Placeholder>) => (
  <Placeholder width="100%" height="19px" {...props} />
))`
  background-color: ${p => p.theme.background};
`;
```

Reply:
@ryan953 I agree this looks better, but the anonymous function will make React Dev Tools a bit sad since it can't show the component name.